backend/db.py

```diff
@@ -279,73 +275,103 @@ def get_member(members: list, member_id: str):
     return {}
 
 
-def sidebar_emitter(
-    org_id, member_id, group_room_name=None
-):  # group_room_name = None or a String of Names
+def sidebar_emitter(org_id:str, member_id:str, group_room_name:str = None) -> dict:  
+    """Function Emits data for the sidebar
```

Comment:
@o4codes , the docstring could be a bit more complete.
**e.g.:**
`Returns:
      A dict mapping keys to the corresponding table row data
      fetched. Each row is represented as a tuple of strings. For
      example:

      {b'Serak': ('Rigel VII', 'Preparer'),
       b'Zim': ('Irk', 'Invader'),
       b'Lrrr': ('Omicron Persei 8', 'Emperor')}

      Returned keys are always bytes.  If a key from the keys argument is
      missing from the dictionary, then that row was not found in the
      table (and require_all_keys must have been False).

    Raises:
      IOError: An error occurred accessing the smalltable.
    """`